src/scripts/move-popup/dimTalentGrid.directive.html

```diff
@@ -1,9 +1,9 @@
-<svg preserveAspectRatio="xMaxYMin meet" svg-bind-viewbox="0 0 {{(vm.numColumns * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor}} {{(vm.numRows * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor + 1}}" class="talent-grid" ng-attr-height="{{(vm.numRows * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor}}" ng-attr-width="{{(vm.numColumns * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor}}">
- <g ng-attr-transform="scale({{vm.scaleFactor}})">
-  <g class="talent-node" ng-attr-transform="translate({{(node.column - vm.hiddenColumns) * (vm.totalNodeSize)}},{{node.row * (vm.totalNodeSize)}})" ng-repeat="node in vm.talentGrid.nodes | talentGridNodes:vm.hiddenColumns track by $index" ng-class="{ 'talent-node-activated': node.activated, 'talent-node-showxp': (!node.activated && node.xpRequired), 'talent-node-default': (node.activated && !node.xpRequired && !node.exclusiveInColumn) }" ng-click="node.hash == 1270552711 ? vm.infuse({'$event': $event}) : true">
-    <circle r="16" cx="-17" cy="17" transform="rotate(-90)" class="talent-node-xp" ng-attr-stroke-width="{{node.xp ? 2 : 0}}" ng-attr-stroke-dasharray="{{100.0 * node.xp / node.xpRequired}} 100" />
+<svg preserveaspectratio="xMaxYMin meet" svg-bind-viewbox="0 0 {{ (vm.numColumns * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor }} {{ (vm.numRows * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor + 1 }}" class="talent-grid" ng-attr-height="{{ (vm.numRows * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor }}" ng-attr-width="{{ (vm.numColumns * vm.totalNodeSize - vm.nodePadding) * vm.scaleFactor }}">
```

Comment:
SVG is particular about the case of attributes - you probably want to keep it `preserveAspectRatio`.